src/Compilers/CSharp/Test/Semantic/SourceGeneration/GeneratorDriverTests.cs

```diff
@@ -2166,6 +2166,69 @@ class C { }
             Assert.Throws<ArgumentNullException>(() => driver.WithUpdatedParseOptions(null!));
         }
 
+        [Fact, WorkItem(57455, "https://github.com/dotnet/roslyn/issues/57455")]
+        public void RemoveTriggeringSyntaxAndVerifySyntaxTreeConsistentWithCompilation()
```

Comment:
>VerifySyntaxTreeConsistentWithCompilation [](http://example.com/codeflow?start=45&length=41)

Are we verifying the syntax tree below, or is this just implicit in the test completing without exceptions?

Reply:
We are verifying the syntax tree given to the SG comes from the correct compilation. So I would say "both".